### PR TITLE
Canary

### PR DESCRIPTION
**Pull request by iruizmar**
Solving #125.

### CODE REVIEW

File: Packages/Core/Runtime/Variables/AtomVariable.cs [hunk 1]
```diff
@@ -47,6 +47,16 @@ public abstract class AtomVariable<T, P, E1, E2, F> : AtomBaseVariable<T>, IGetE
...
 
+        /// <summary>
+        /// Whether Changed Event should be triggered on OnEnable or not
+        /// </summary>
+        public bool TriggerChangedOnOnEnable;
+
+        /// <summary>
+        /// Whether ChangedWithHistory Event should be triggered on OnEnable or not
+        /// </summary>
+        public bool TriggerChangedWithHistoryOnOnEnable;
```
Comment (AdamRamberg) [inline Packages/Core/Runtime/Variables/AtomVariable.cs:58]: Make the variable private, rename it to `_triggerChangedOnOnEnable` and add the attribute `[SerializeField]`.

File: Packages/Core/Runtime/Variables/AtomVariable.cs [hunk 2]
```diff
@@ -47,6 +47,16 @@ public abstract class AtomVariable<T, P, E1, E2, F> : AtomBaseVariable<T>, IGetE
         /// </summary>
         public E2 ChangedWithHistory;
 
+        /// <summary>
+        /// Whether Changed Event should be triggered on OnEnable or not
+        /// </summary>
+        public bool TriggerChangedOnOnEnable;
```
Comment (AdamRamberg) [inline Packages/Core/Runtime/Variables/AtomVariable.cs:53]: Make the variable private, rename it to `_triggerChangedOnOnEnable ` and add the attribute `[SerializeField]`.

File: Packages/Core/Runtime/Variables/AtomVariable.cs (hunk 1 above)
Comment (AdamRamberg) [reply Packages/Core/Runtime/Variables/AtomVariable.cs:58]: Forgot initially to add that for clarity sake you should also explicitly set the value to `default` or `false`.

File: Packages/Core/Runtime/Variables/AtomVariable.cs (hunk 2 above)
Comment (AdamRamberg) [reply Packages/Core/Runtime/Variables/AtomVariable.cs:53]: Forgot initially to add that for clarity sake you should also explicitly set the value to `default` or `false`.

**Comment by iruizmar** [comment]
I forgot to leave a comment. I made the requested changes :). Let me know if there's something else.

**Comment by AdamRamberg** [comment]
Looks good now! Merging the change to canary.

**Pull request merged**